jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -54,12 +54,23 @@ def functionTest(String test_name, String TEST_GROUP, Boolean RUN_FIT_TEST, Bool
             try{
                 timeout(90){
                     // run test script
-                    sh '''#!/bin/bash -x
-                    pushd build-config
-                    ./build-config
-                    popd
-                    ./build-config/test.sh
-                    '''
+                    def repos = env.REPOS_UNDER_TEST.tokenize(',')
```

Comment:
@PengTian0 The continuous test passed in Sandbox, with a few regression issues that are irrelavent to my code change if it runs during the fixed period of time. So the problem itself is fixed. Passed from Build 1083 to 1092, while from Build 1092 on, the case failed due to no active poller which is irrelavent to my code change. Need others involvement to take a look.
http://10.62.59.175:8080/job/ContinuousTest/1092/